client/src/components/layout/BuildsList.js

```diff
@@ -12,11 +13,11 @@ const BuildsList = (props) => {
         } catch (error) {
             console.error(`Error Fetching Build List: ${error}`)
         }
-    }
+    }    
 
     const buildTiles = currentBuilds.map((build) => {
         return (
-            <BuildTile key={build.id} build={build} />
+            <li key={build.id}><BuildTile build={build} /></li>
```

Comment:
if you want a `li` it belongs in the BuildTile component where the other HTML styling is set up for each tile